src/main/resources/static/css/box.css

```diff
@@ -8,4 +8,10 @@
     display: flex;
     justify-content: center;
     margin-top: 90px;
+}
+
+.site-footer {
```

Comment:
as this is the same as the site-header you can write it like this:
`
.site-header, .site-footer {
    background-color: #1e194b;
    -webkit-backdrop-filter: saturate(180%) blur(20px);
    backdrop-filter: saturate(180%) blur(20px);
}
`

Reply:
you resolved this but didn't do it? see above (where the site-header is, line 1)

Reply:
done